src/pages/DemographicsPage/index.jsx

```diff
@@ -153,9 +159,11 @@ export default function DemographicsPage({ navSearchSearching, motto }) {
             />
             <CardHIP
               className={'blue six-tile house-type'}
-              heading={'House Type'}
+              heading={'Household Composition'}
+              chartType={'donut'}
+              dataResponse={houseData}
               secondaryInfo={
-                "Here's what the makeup of houses tend to look like"
+                `Hover over the chart slices to find out the different type of occupants you can expect to find in the borough of ${houseData.borough_name}`
```

Comment:
Should "type of" be "types of"? Also consider using households instead of occupants.